src/rgw/rgw_rest_swift.cc

```diff
@@ -777,11 +777,20 @@ int RGWGetObj_ObjStore_SWIFT::send_response_data(bufferlist& bl, off_t bl_ofs, o
 {
   string content_type;
 
-  if (sent_header)
+  if (sent_header) {
```

Comment:
@oritwas ensuring that _send_response_data()_ is called after performing all operations which could result in HTTP error looks to me like a caller responsibility.
Without the check we would send second HTTP header for the same request.